notebook/notebookapp.py

```diff
@@ -71,10 +68,7 @@
 )
 
 # py23 compatibility
-try:
-    raw_input = raw_input
-except NameError:
-    raw_input = input
+raw_input = input
```

Comment:
It looks like this is no longer used, so it should just go away entirely.

Reply:
Hi, @takluyver. You are right. I have removed it.